lib/dor/assembly/content_metadata.rb

```diff
@@ -49,19 +44,6 @@ def self.find_common_path(objects)
         ::Assembly::ObjectFile.common_path(all_paths) # find common paths to all files provided if needed
       end
       private_class_method :find_common_path
-
-      def self.object_level_type(style)
-        raise "Supplied style (#{style}) not valid" unless VALID_STYLES.include? style
```

Comment:
doesn't this PR lose this validity check?

Reply:
Yes, but the check is unnecessary as the inputs are limited by what is provided in this repo.